controllers/controllers.go

```diff
@@ -0,0 +1,33 @@
+package controllers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/JoseLuisTorrentera/academy-go-q42021/repository"
```

Comment:
Use best practice, separate your package from third parties. The Convension should be:
Import(
Core packages

Your packages

Third party packages)